src/aura/GE_GiftEntry/GE_GiftEntryController.js

```diff
@@ -1,4 +1,35 @@
 ({
+    
+    doInit: function(cmp) {
+        var action = cmp.get("c.createDefaultTemplate");
```

Comment:
This is probably late in the game to share this thought, but what if we moved this call into the error handler for where we try to load the default template (on SGE). That way we could avoid the querying etc until we know we actually need it. (This probably contradicts my earlier thoughts on how we deliver this, so maybe this isn't a critical change at this stage)